src/main/java/com/denys/toukdemo/entity/RoomEntity.java

```diff
@@ -0,0 +1,78 @@
+package com.denys.toukdemo.entity;
+
+import com.denys.toukdemo.dto.RoomType;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+@Entity
+@Table(name = "room", schema = "public", catalog = "postgres")
```

Comment:
po co tak specyficznie? sama definicja nazwy tabeli nie wystarczy?

Reply:
> po co tak specyficznie? sama definicja nazwy tabeli nie wystarczy?

skorzystałem z pluginu w Intellij do generowania Entity 
wystarczy, coś podobne raczej jest potrzebne w przypadku dostępu to tabel w przypadku kilku schem/katalogów w postgres